### PR TITLE
Fix 3192

### LINKED ISSUE #3192: [Bug] `crypto/dd/twitter` errors
**Describe the bug**
1. `--limit` flag not available in dropdown and only `-` options available
Screen Shot 2022-10-30 at 2 45 36 PM

2. `--descend` flag not working

**To Reproduce**
Steps(from the start) and commands to reproduce the behavior

### PR DESCRIPTION
**Pull request by jose-donato**
# Description

Fix #3192 

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py [hunk 1]
```diff
@@ -176,8 +176,12 @@ def __init__(
                 "--descend": {},
             }
             choices["events"]["-s"] = {c: None for c in coinpaprika_view.EVENTS_FILTERS}
-            choices["twitter"]["-s"] = {
-                c: None for c in coinpaprika_view.TWEETS_FILTERS
+            choices["twitter"] = {
+                "--sort": {c: {} for c in coinpaprika_view.EX_FILTERS},
```
Comment (hjoaquim) [inline openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:180]: Why the change from `TWEETS_FILTERS` to `EX_FILTERS`?

image

image

Looking at the df seems we want the `TWEETS_FILTERS`:
image

**Review by hjoaquim — CHANGES_REQUESTED** [review]
Other than this, it's alright!

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py (hunk 1 above)
Comment (jose-donato) [reply openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:180]: Yessirrr my bad. Thanks for the review

**Review by hjoaquim — APPROVED** [review]
:bug: :skull_and_crossbones:
